components/gcds-card/mws-no-workaround-fr.html

```diff
@@ -0,0 +1,71 @@
+---
+dateModified: "2025-07-04"
+description: "Pilote de co-existance de la carte GCDS dans GCWeb tel qu'implémenté dans le système web géré."
+language: "fr"
+altLangPage: "mws-no-workaround-en.html"
```

Comment:
```suggestion
altLangPage: "mws-no-workaround-fr.html"
```

Reply:
This don't seem to be an issue. The page language switch does work as expected.